test/test_distributions.py

```diff
@@ -2623,6 +2623,37 @@ def test_kl_monte_carlo(self):
                 'Actual (analytic): {}'.format(actual),
             ]))
 
+    # Multivariate normal has a separate Monte Carlo based test due to the requirement of random generation of
+    # positive (semi) definite matrices. n is set to 5, but can be increased during testing.
+    def test_kl_multivariate_normal(self):
```

Comment:
Consider adding another simple test of batched computation, something like
```py
def test_kl_multivariate_normal_batched(self):
    loc = [torch.randn(7, 3) for _ in range(2)]
    scale_tril = [transform_to(constraints.lower_cholesky)(torch.randn(7,3,3))
                  for _ in range(2)]
    expected_kl = torch.cat([
        kl_divergence(MultivariateNormal(locs[0][i], scale_tril[0][i]),
                      MultivariateNormal(locs[1][i], scale_tril[1][i]))
        for i in range(7)])
    actual_kl = kl_divergence(MultivariateNormal(locs[0], scale_tril[0]),
                              MultivariateNormal(locs[1], scale_tril[1]))
    self.assertEqual(actual_kl, expected_kl)
```

Reply:
Sure.